docs/aws/audit/sesmonitoring/rules/is_identity_verified_remediation.mdx

```diff
@@ -1,6 +1,220 @@
+
+By following these steps, you can prevent the misconfiguration of unverified identities in AWS SES using Python scripts.
+</Accordion>
+
+</AccordionGroup>
+</Tab>
+<Tab title='Cause'>
+### Check Cause
+<AccordionGroup>
+<Accordion title='Using Console' defaultOpen='true'>
+1. Sign in to the AWS Management Console.
+2. Navigate to the Simple Email Service (SES) dashboard by typing 'SES' into the search bar and selecting 'Simple Email Service'.
+3. In the left-hand navigation pane, click on 'Identity Management', then select 'Email Identities' or 'Domain Identities' based on your requirement.
+4. Here, you can see the list of all the identities (email addresses and domains). Check the 'Verification Status' column for each identity. If the status is 'verified', then the identity is correctly configured. If the status is 'pending verification' or 'failed', then the identity is not verified, indicating a misconfiguration.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local machine and configure it with your AWS account credentials. You can do this by running the following commands:
+
+   Installation:
+   ```
+   pip install awscli
+   ```
+   Configuration:
+   ```
+   aws configure
+   ```
+   You will be prompted to provide your AWS Access Key ID, Secret Access Key, Default region name, and Default output format.
+
+2. List all SES identities: Once your AWS CLI is set up, you can list all the Simple Email Service (SES) identities in your account by running the following command:
+   ```
+   aws ses list-identities
+   ```
+   This command will return a list of all identities (email addresses and domains) that you have attempted to verify.
+
+3. Check the verification status of each identity: For each identity listed, you can check its verification status by running the following command:
+   ```
+   aws ses get-identity-verification-attributes --identities "identity1" "identity2" "identity3"
+   ```
+   Replace "identity1", "identity2", "identity3" with the actual identities you want to check. This command will return the verification status for each identity.
+
+4. Analyze the output: The output of the previous command will show the verification status of each identity. If the verification status is "Pending", it means the identity is not verified. If the verification status is "Success", it means the identity is verified.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary AWS SDK for Python (Boto3) if you haven't done so already. You can do this by running the command `pip install boto3` in your terminal.
+
+2. Import the necessary modules and create a session using your AWS credentials. Here's a basic example:
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>